File: javascript/recursive_count.js
function recursiveCount(num = 0) {
  if (num === 10) return num;
  console.log(num);
  recursiveCount(num + 1);
}

if (require.main === module) {
  recursiveCount();
}

module.exports = recursiveCount;

// OPTIONAL
// Please add your pseudocode to this file

// return if count = 10
// count += 1
// count()

// And a written explanation of your solution

File: ruby/recursive_count.rb
def recursive_count(num = 0)
  if count < 10
    puts count
    recursive_count(num + 1)
  end

end

if __FILE__ == $PROGRAM_NAME
  recursive_count
end

# OPTIONAL
# Please add your pseudocode to this file
# And a written explanation of your solution
